src/algebra/big_operators/finprod.lean

```diff
@@ -87,13 +87,13 @@ open_locale classical
 /-- Sum of `f x` as `x` ranges over the elements of the support of `f`, if it's finite. Zero
 otherwise. -/
 @[irreducible] noncomputable def finsum {M α} [add_comm_monoid M] (f : α → M) : M :=
-if h : finite (support (f ∘ plift.down)) then ∑ i in h.to_finset, f i.down else 0
+if h : (support (f ∘ plift.down)).finite then ∑ i in h.to_finset, f i.down else 0
```

Comment:
For the sake of clarity on my part, this change is needed because `finite` no longer typechecks here, right? And you could have instead changed it to `set.finite (support ...)` (not that you should have!)

Reply:
Indeed, that's what's going on (in particular, `error: unknown identifier 'finite'`). I generally switched things over to dot notation when it worked, but I used `set.finite` when dot notation didn't work due to elaboration order.